app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'user_id'
```

Comment:
- [ ] As requested by the previous code reviewer, which is a requirement for this project, the likes belongs to an author through the author_id, not user_id. Kindly remove foreign_key: 'user_id' and replace it with author_id.